charts/observability-pipeline/templates/_helpers.tpl

```diff
@@ -109,7 +109,7 @@ agent:
   executable: /otelcol-contrib
   description:
     identifying_attributes:
-      service.name: collector
+      service.name: {{ .Values.collector.serviceName }}
```

Comment:
I don't think beekeeper currently supports custom service.name for its agents. https://github.com/honeycombio/beekeeper/blob/05cac45b6e09f529f065e72e793cfa28e8e640e4/internal/opamp/agent_type.go#L22

Reply:
Ah, so if the chart makes this configurable we could break interaction with Beekeeper?

Reply:
Yea, I believe beekeeper rely on this to differentiate which config sent to which agent